test/testcil.pl

```diff
@@ -469,8 +469,8 @@ sub addToGroup {
                         "Notbug. Should fail since we don't pad argument lists");
 addTest("test/restrict EXTRAARGS=-std=c9x _GNUCC=1");
 addTest("test/restrict1 _GNUCC=1");
-addTest("testrun/rmtmps1 ");
-addTest("testrun/rmtmps2 _GNUCC=1");
+addTest("testrun/rmUnused1 ");
+addTest("testrun/rmUnused2 _GNUCC=1");
```

Comment:
Why does the name of the test matter here? That seems odd?

Reply:
I renamed the C file that contains the test: [test/small1/rmtmps1.c → test/small1/rmUnused1.c](https://github.com/goblint/cil/pull/135/files#diff-0723c1182ff96b92335bc41cd0ea94c133f2a9382b13b9b5bb0661940a45ccdf). Not strictly necessary, but keeps things consistent.

(In my first commit, I forgot to change the addTest call, hence the second commit.)